Loenn/entities/arrowBubble.lua

```diff
@@ -0,0 +1,31 @@
+local arrowBubble = {}
+
+arrowBubble.name = "isaBag/arrowBubble"
+arrowBubble.depth = -100
+arrowBubble.placements = {
+    {
+        name = "Arrow Bubble (Left, IsaGrabBag)",
```

Comment:
to follow up on catapillie's comment, you would change this line to be `name = "left"` and then have `[...]placements.name.left = "Arrow Bubble (Left)"` in the en_gb.lang. check out some of the other legacy loenn plugin repos (or vanilla) for examples